src/pages/api/auth/submit-code.ts

```diff
@@ -1,7 +1,11 @@
 import type { APIRoute } from 'astro'
-import { getSession, getUserByEmail, makeOrUpdateSession, findDocument } from '../../../lib/game-saving/account'
+import { getSession, getUserByEmail, makeOrUpdateSession, findDocument, updateDocument } from '../../../lib/game-saving/account'
 import { isValidEmail } from '../../../lib/game-saving/email'
 import {DevEmail} from "../../../lib/game-saving/auth-helper";
+import { Timestamp } from 'firebase-admin/firestore';
+
+const MAX_ATTEMPTS = 3;
+const LOCKOUT_DURATION_MS = 15 * 60 * 1000;
```

Comment:
let's make this an environment variable so that this will be easier to change in the future